src/cueUtilities.js

```diff
@@ -319,60 +320,83 @@ module.exports = function (params, cy, api) {
           }
         });
 
-        cy.on('tap', 'node', data.eTap = function (event) {
-          var node = this;
-          var opts = options();
-          let nodeHasCue = isSelectedGroupsContains(node);
-          if (hoveredGroup) {
-            nodeHasCue = nodeHasCue || hoveredGroup.id() === node.id();
+        function eventOnCueRegion(node, event) {
```

Comment:
My oversight from earlier - since function returns a boolean, name should start with `is`.